composer.json

```diff
@@ -25,7 +25,7 @@
   ],
   "require" : {
     "php" : ">=7.1",
-    "oat-sa/tao-core" : ">=50.5.0||dev-develop",
+    "oat-sa/tao-core" : "dev-feature/AUT-2995",
```

Comment:
This repository is deprecated - not used anymore. I am not sure we need this change here.